tailwind.config.js

```diff
@@ -1,12 +1,15 @@
 /** @type {import('tailwindcss').Config} */
 module.exports = {
-	content: ["./src/**/*.{astro,html,jsx,tsx,svelte,vue,js,ts}"],
+	content: [
+		"./public/**/*.html",
```

Comment:
I don't this this change is needed, by convention all of our views should live inside of `/src`. If someone needs to support `/public` then they should add this key for their use case.